sematic/ui/src/index.tsx

```diff
@@ -215,6 +215,7 @@ function Router() {
 
         // Use obfuscated host name
         currentUrl.hostname = currentHostName;
+        currentUrl.pathname = '[removed]';
```

Comment:
Does this need to be set to a string? Can't it just be null?

Reply:
+1

Reply:
I intentionally did that to differentiate the cases of a. "there was not a pathname"  vs b. "there was a pathname but intentionally removed". So when preview the data in the data platform, it is clear to know what happened and how.

I don't feel strongly. But if you do believe setting it to null is better, please let me know.

Reply:
I expect when there is no path, the value to be `''`. If so, you can use `null` for when it was removed.